BlueprintUI/Sources/BlueprintView/BlueprintView.swift

```diff
@@ -403,7 +434,7 @@ extension BlueprintView {
                             contentView.insertSubview(controller.view, at: index)
                         }
 
-                        controller.update(node: child, appearanceTransitionsEnabled: true)
+                        controller.update(node: child, context: context)
```

Comment:
Passing `true` all the time here was wrong; but also a no-op kind of wrong; since we'd never have any children to update if we hadn't done at least one update path.